blosc2/CMakeLists.txt

```
@@ -18,13 +25,9 @@ else()
     set(BLOSC_INSTALL ON)
     add_subdirectory(c-blosc2)
     include_directories("${CMAKE_CURRENT_SOURCE_DIR}/c-blosc2/include")
-    target_link_libraries(blosc2_ext blosc2_static)
+    target_link_libraries(blosc2_ext PRIVATE blosc2_static Python::Numpy)
```

```suggestion
    target_link_libraries(blosc2_ext PRIVATE blosc2_static Python::NumPy)
```

Case sensitive, otherwise, you can put it anywhere anywhere in the script

---

Thanks Cristian!